Source/WebCore/css/html.css

```diff
@@ -316,31 +314,34 @@ ol {
     list-style-type: decimal;
     margin-block-start: 1__qem;
     margin-block-end: 1em;
-    margin-inline-start: 0;
-    margin-inline-end: 0;
     padding-inline-start: 40px;
 }
 
+dir {
+    display: block;
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+}
+
```

Comment:
Can we remove this block? It's covered by the previous rule, with the difference that the previous rule is using __qem

Reply:
Good spot!